p2p/types/network_peer.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+import (
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+)
+
+type NetworkPeer struct {
+	Dialer     Transport
```

Comment:
Can we lowercase (i.e. not expose outside this package) the fields?

Reply:
Not really, these fields are being accessed directly in multiple places outside of this package